### ISSUE TITLE
Backup Format compatibility with Skytube

### ISSUE THREAD

**Issue by ff1992ff** [TeamNewPipe/NewPipe#4445]
### Checklist

- [x ] I checked, but didn't find any duplicates of this issue in the repo. 
- [x ] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x ] This issue contains only one feature request. I will open one issue for every feature I want to request.

Hey guys,
I really like the new update - Newpipe has become so much better. 

My feature wish might be a bit controversial since another project is involved. I am using Skytube and Newpipe - switching between with regards to which current version has the better features. 

The thing that for me is a bit annoying is, when migrating channels from one app to the other. I have to do this by hand, because the channel backup format between both apps is incompatible. 

I would appreciate it, if either a new open format for channel backups would be created in consultation with team skytube or if the decoding logic for  skytube backups would be implemented in future versions. 

Thank you for your consideration and for the great work you are doing!

**Comment by AudricV** [comment]
Seeing how the SkyTube developers are currently active (no app commits since 6 months at the time of writing, lot of issues without an answer) and our capabilities, a collaboration with the SkyTube team will not happen I am afraid, especially as we already have a lot of things to do before starting the app rewrite.

However, multiple "Tube" projects like NewPipe have started a while ago to discuss about common formats: see https://github.com/UniversalPipeWrench/unified-user-data-format/. Your answer about a new open format for channel backups will be probably here, in the long term.

Closing this issue as wontfix.
